code/API_definitions/location-verification.yaml

```diff
@@ -456,6 +458,19 @@ components:
             status: 404
             code: NOT_FOUND
             message: "The specified resource is not found"
+    VerifyLocationUnprocessableEntity422:
```

Comment:
This is in contradiction with L38:
maxAge=0 means a fresh calculation is requested by the client. If the system does not support it, or fresh location cannot be checked at that time for any reason, the API response will be "UNKNOWN" and the lastLocationTime attribute may indicate the last available time for the device location.

The is also an ongoing discussion for this in the related issue

Reply:
Decision during last meeting was to apply 422 to verification as well, but we have to update description (L38) accordingly